src/views/LimitedRestorationExtension.vue

```diff
@@ -194,6 +184,12 @@ export default class LimitedRestorationExtension extends Vue {
   /** The restoration filing ID. */
   @Prop({ default: 0 }) readonly restorationId!: number
 
+  /**
+   * "isDataLoaded" is a flag that is to "true" after all data is loaded.
+   * This is to prevent using the state filing restoration before it's fetched.
+   */
+  isDataLoaded = false
```

Comment:
This was a huge problem for me when I was doing the work. I'll be keeping this way in mind moving down the road.

Reply:
To be clear, this is a hack. A better implementation is for sub-components to handle the situation of "no data" initially and "new data" when it comes in.

Reply:
That makes sense. OK, noted Sev. Thanks!